.github/pull_request_template.md

```diff
@@ -0,0 +1,10 @@
+
+### Description
+
+### Documentation
+
+If enhancement: provide description with example code/config snippet or pointer to issue with the description
```

Comment:
Should these be in pseudo Markdown comments similar to what we do for the issue template ?
These can be cumbersome as sometime people edit the comments directly.

E.g.
```
[//]: # "enhancement: provide example code/config snippet or pointer to issue with the same"
[//]: # "feature: documentation changes are included in the PR or provide a pointer to a documentation issue/PR"
[//]: # "others: use "None" if the PR has no impact on documentation
```